src/interface/read_config_file.py

```diff
@@ -0,0 +1,108 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
+    
+    def __init__(self, fileconfig): 
+            
+            
+            self.xmld = etree.parse(fileconfig)             
+            self.root = self.xmld.getroot()
+
+        
+
+    def get_output_file(self):
+        """ 
+        Returns the outputfile path, name and extension values from config file
+        listoutput: variable to assign all values read from config file
```

Comment:
the parameter name is not the same that the one used in the method, what kind of value is? array, string, object, etc, at least you should provide an example of the return value